src/screens/FeedScreen/hooks/useFeedHeader.js

```diff
@@ -71,7 +71,12 @@ const useFeedHeader = ({actor, source}) => {
     if (source && id) sendViewTimePost(id);
 
     resetTimer();
-    navigation.goBack();
+    navigation.navigate('HomeTabs', {
+      screen: 'Feed',
+      params: {
+        isGoBack: true
+      }
+    });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `navigation.goBack()` to `navigation.navigate('HomeTabs', { screen: 'Feed', params: { isGoBack: true } })` seems to be a significant one. It's not just going back to the previous screen, but navigating to a specific screen ('Feed') in the 'HomeTabs' stack with an additional parameter `isGoBack: true`. 

This could potentially introduce unexpected behavior if other parts of the codebase are expecting the navigation stack to behave in a certain way after this function is called. 

Ensure that all parts of the application that interact with this navigation stack are updated to handle this new navigation logic correctly. Also, make sure to test thoroughly for any regressions this might cause in your app's navigation flow.


<!-- This is an auto-generated comment by OSS CodeRabbit -->